### PR TITLE
Port System.ComponentModel.Composition source code comments to Docs

### PR DESCRIPTION
**Pull request by carlossanlop**
## Summary

Automatically ported exceptions found in triple slash source code comments in System.ComponentModel.Composition that did not exist in dotnet-api-docs.

Fixes #Issue_Number (if available)

### CODE REVIEW

**Comment by carlossanlop** [comment]
@mairaw @rpetrusha please take a look whenever you get a chance.
Adding area owners @maryamariyan, @safern in case they have any suggestions or comments.

File: xml/System.ComponentModel.Composition/CompositionContractMismatchException.xml [hunk 1]
```diff
@@ -150,6 +150,9 @@
  ]]></format>
         </remarks>
         <related type="Article" href="~/docs/standard/serialization/xml-and-soap-serialization.md">XML and Soap Serialization</related>
+        <exception cref="T:System.ArgumentNullException"><paramref name="info" /> is <see langword="null" />.</exception>
+        <exception cref="T:System.Runtime.Serialization.SerializationException"><paramref name="info" /> is missing a required value.</exception>
+        <exception cref="T:System.InvalidCastException"><paramref name="info" /> contains a value that cannot be cast to the correct type.</exception>
```
Comment (safern) [inline xml/System.ComponentModel.Composition/CompositionContractMismatchException.xml]: cast -> casted ?

Comment (carlossanlop) [reply xml/System.ComponentModel.Composition/CompositionContractMismatchException.xml]: Good question. Apparently both words are correct for conjugating in past tense: https://en.bab.la/conjugation/english/cast

Comment (rpetrusha) [reply xml/System.ComponentModel.Composition/CompositionContractMismatchException.xml]: "cast" is both the present and past tense of the verb.

File: xml/System.ComponentModel.Composition/AttributedModelServices.xml [hunk 2]
```diff
@@ -745,6 +745,9 @@
   
  ]]></format>
         </remarks>
+        <exception cref="T:System.ArgumentNullException"><paramref name="compositionService" /> or <paramref name="attributedPart" /> is <see langword="null" />.</exception>
+        <exception cref="T:System.ComponentModel.Composition.CompositionException">An error occurred during composition. <see cref="P:System.ComponentModel.Composition.CompositionException.Errors" /> will contain a collection of errors that occurred.</exception>
```
Comment (rpetrusha) [inline xml/System.ComponentModel.Composition/AttributedModelServices.xml]:
```suggestion
        <exception cref="T:System.ComponentModel.Composition.CompositionException">An error occurred during composition. <see cref="P:System.ComponentModel.Composition.CompositionException.Errors" /> contains a collection of errors that occurred.</exception>
```

File: xml/System.ComponentModel.Composition/AttributedModelServices.xml [hunk 3]
```diff
@@ -776,6 +779,8 @@
         <remarks>To be added.</remarks>
         <exception cref="T:System.ArgumentNullException">
           <paramref name="reflectionContext" /> is <see langword="null" />.</exception>
+        <exception cref="T:System.ComponentModel.Composition.CompositionException">An error occurred during composition. <see cref="P:System.ComponentModel.Composition.CompositionException.Errors" /> will contain a collection of errors that occurred.</exception>
```
Comment (rpetrusha) [inline xml/System.ComponentModel.Composition/AttributedModelServices.xml]:
```suggestion
        <exception cref="T:System.ComponentModel.Composition.CompositionException">An error occurred during composition. <see cref="P:System.ComponentModel.Composition.CompositionException.Errors" /> contains a collection of errors that occurred.</exception>
```

File: xml/System.ComponentModel.Composition/AttributedModelServices.xml [hunk 4]
```diff
@@ -776,6 +779,8 @@
...
+        <exception cref="T:System.ObjectDisposedException">The <see cref="T:System.ComponentModel.Composition.ICompositionService" /> has been disposed of.</exception>
```
Comment (rpetrusha) [inline xml/System.ComponentModel.Composition/AttributedModelServices.xml]:
```suggestion
        <exception cref="T:System.ObjectDisposedException">The <see cref="T:System.ComponentModel.Composition.ICompositionService" /> has been disposed.</exception>
```

File: xml/System.ComponentModel.Composition/AttributedModelServices.xml [hunk 5]
```diff
@@ -745,6 +745,9 @@
   
  ]]></format>
         </remarks>
+        <exception cref="T:System.ArgumentNullException"><paramref name="compositionService" /> or <paramref name="attributedPart" /> is <see langword="null" />.</exception>
+        <exception cref="T:System.ComponentModel.Composition.CompositionException">An error occurred during composition. <see cref="P:System.ComponentModel.Composition.CompositionException.Errors" /> will contain a collection of errors that occurred.</exception>
+        <exception cref="T:System.ObjectDisposedException">The <see cref="T:System.ComponentModel.Composition.ICompositionService" /> has been disposed of.</exception>
```
Comment (rpetrusha) [inline xml/System.ComponentModel.Composition/AttributedModelServices.xml]:
```suggestion
        <exception cref="T:System.ObjectDisposedException">The <see cref="T:System.ComponentModel.Composition.ICompositionService" /> has been disposed.</exception>
```

File: xml/System.ComponentModel.Composition/ICompositionService.xml [hunk 6]
```diff
@@ -39,6 +39,9 @@
         <param name="part">The part to compose.</param>
         <summary>Composes the specified part, with recomposition and validation disabled.</summary>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentNullException"><paramref name="part" /> is <see langword="null" />.</exception>
+        <exception cref="T:System.ComponentModel.Composition.CompositionException">An error occurred during composition. <see cref="P:System.ComponentModel.Composition.CompositionException.Errors" /> will contain a collection of errors that occurred.</exception>
```
Comment (rpetrusha) [inline xml/System.ComponentModel.Composition/ICompositionService.xml]:
```suggestion
        <exception cref="T:System.ComponentModel.Composition.CompositionException">An error occurred during composition. <see cref="P:System.ComponentModel.Composition.CompositionException.Errors" /> contains a collection of errors that occurred.</exception>
```

File: xml/System.ComponentModel.Composition/ICompositionService.xml [hunk 7]
```diff
@@ -39,6 +39,9 @@
...
+        <exception cref="T:System.ObjectDisposedException">The <see cref="T:System.ComponentModel.Composition.ICompositionService" /> has been disposed of.</exception>
```
Comment (rpetrusha) [inline xml/System.ComponentModel.Composition/ICompositionService.xml]:
```suggestion
        <exception cref="T:System.ObjectDisposedException">The <see cref="T:System.ComponentModel.Composition.ICompositionService" /> has been disposed.</exception>
```

File: xml/System.ComponentModel.Composition/CompositionContractMismatchException.xml (hunk 1 above)
Comment (carlossanlop) [reply xml/System.ComponentModel.Composition/CompositionContractMismatchException.xml]: Then it will remain unchanged.

**Comment by carlossanlop** [comment]
@rpetrusha I think I addressed all the suggestions. Any chance we can merge this?

**Comment by rpetrusha** [comment]
Yes, I'll merge it now, @carlossanlop. Thanks for making the additional changes.

**Pull request merged**